core/discovery/proposal/filter_test.go

```diff
@@ -19,12 +19,15 @@ package proposal
 
 import (
 	"testing"
+	"time"
 
 	"github.com/mysteriumnetwork/node/market"
 	"github.com/mysteriumnetwork/node/money"
 	"github.com/stretchr/testify/assert"
 )
 
+const bytesInGigabyte = 1000000000
```

Comment:
I also love zeros, but:
```suggestion
const bytesInGigabyte = 1_000_000_000
```

Also, I don't get the discrepancy between this and `datasize.BitSize`?

Reply:
bitsize is exactly this - bits, we're not counting bits here.

Reply:
I was referring to the logic in the given package, where binary conversion is used instead of decimal SI:
`Kilobyte = 1024 * Byte`

Reply:
What I meant is you should either use it or change it, but don't create 2 different versions of data counting - it will create discrepancies if data is interpreted in different ways throughout the app.

Reply:
We can change this in a separate PR.